src/general/PauseMenu.cs

```diff
@@ -71,6 +71,9 @@ public override void _Ready()
 
     public override void _UnhandledInput(InputEvent @event)
     {
+        bool noExclusiveTutorialActive =
+            GameProperties.TutorialState == null || !GameProperties.TutorialState.ExclusiveTutorialActive();
```

Comment:
ExclusiveTutorialActive can end up running quite a few checks, so maybe using a local function here would be better to not have to run that check for each key press. Instead the check would be done just before needing the information.